### ISSUE TITLE
Export/Import Saves

### ISSUE THREAD

**Issue by jawad-kp** [allen505/save-for-later-firefox#28]
An option to export or import tabs/saves from a file. Apart from the ability to back-up settings when we switch machines, it would also let you share your saves with people.

**Comment by allen505** [comment]
Hey @jawad-kp , this looks similar to a request by @AzDecker mentioned [here](https://github.com/allen505/save-for-later/issues/15#issuecomment-1465409966) which in turn could be solved by https://github.com/allen505/save-for-later/issues/3. Would it solve your use case too?

**Comment by davidp88** [comment]
export all the settings and tabs and import all,useful when moving addon to new PC

**Comment by allen505** [comment]
Hi @davidp88. The reason I mentioned #3 is because we plan to move the storage from local to sync and that will allow your saves to be in Sync across all your devices (via your browser's account)
But yes, if folks do not sign in to their accounts in browsers, then exporting would be useful for them

**Comment by jawad-kp** [comment]
> Hey @jawad-kp , this looks similar to a request by @AzDecker mentioned [here](https://github.com/allen505/save-for-later/issues/15#issuecomment-1465409966) which in turn could be solved by #3. Would it solve your use case too?

That solves a part of it. You may also want to cover people switching browsers. But I get that it's not a gigantic use case.

**Comment by allen505** [comment]
Yeah, that's a valid use case, but not a priority. So, I'll keep this issue open
